### PR TITLE
functioning logout button with logout toast added

### PR DESCRIPTION
**Pull request by zsofinnemeth**
(This may not really belong to this branch but we did not want to start a new branch for this small feature)

### CODE REVIEW

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 1]
```diff
@@ -27,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);
         setSupportActionBar(myToolbar);
         textViewWelcome = (TextView) findViewById(R.id.textViewWelcome);
+        logoutButton =(Button) findViewById(R.id.logoutButton);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth pls move this into the options menu

Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: moved to the options menu of the toolbar

Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @adamgyulavari

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 2]
```diff
@@ -11,13 +11,16 @@
...
 import android.view.MenuInflater;
 import android.view.MenuItem;
+import android.widget.Button;
 import android.widget.Toast;
 
 import com.greenfox.fuchsit.zerdareader.R;
 
 public class MainActivity extends AppCompatActivity {
 
+    Button logoutButton;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth you don't use this variable

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 3]
```diff
@@ -49,6 +52,17 @@ private void checkIfLoggedIn() {
         }
     }
 
+    public void logOut(){
+       SharedPreferences loginData = PreferenceManager.getDefaultSharedPreferences(this);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth you already have a sharedpreferences reference, you don't need another.

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 4]
```diff
@@ -49,6 +52,17 @@ private void checkIfLoggedIn() {
...
+       SharedPreferences.Editor editor = loginData.edit();
+        editor.putString("userName", "");
+        editor.putString("password", "");
+        editor.apply();
+        Toast.makeText(this,"Successful logout", Toast.LENGTH_SHORT).show();
+        Intent loginIntent = new Intent(MainActivity.this, LoginActivity.class);
+        startActivity(loginIntent);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:61]: @zsofinnemeth pay attention to indentation!

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 3 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: sorry badly handled merge conflict.. in a minute

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 4 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:61]: @adamgyulavari  I hope I got it right by now!

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java [hunk 5]
```diff
@@ -42,13 +42,25 @@ protected void onCreate(Bundle savedInstanceState) {
...
             startActivity(i);
             finish();
         }
     }
 
+    public void logOut() {
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+            editor.putString("userName", "");
+            editor.putString("password", "");
+            editor.apply();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @zsofinnemeth almost :) these 3 lines are not in a new block, so they should be indented as the same as the above editor creation. Also resolve conflicts and then this PR is mergable.

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java (hunk 5 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java]: @adamgyulavari  fixed! (?)

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**